.gitignore

```diff
@@ -12,9 +12,10 @@ deployments/hardhat
 deployments/local
 deployments/devchain
 typechain-types
-
 # IDEA
 .idea
+.env.staging
```

Comment:
Agree with @soloseng, add a `# decrypted environment configs` section, right now looks like these are related to IDEA.